pkg/kube/snapshot/snapshot.go

```diff
@@ -84,32 +82,30 @@ type Snapshotter interface {
 	//
 	// 'name' is the name of the VolumeSnapshot that will be cloned.
 	// 'namespace' is the namespace of the VolumeSnapshot that will be cloned.
-	// 'cloneName' is name of the clone.
-	// 'cloneNamespace' is the namespace where the clone will be created.
 	// 'waitForReady' will make the function blocks until the clone's status is ready to use.
-	// 'labels' is the labels to set on the created VSC
-	Clone(ctx context.Context, name, namespace, cloneName, cloneNamespace string, waitForReady bool, labels map[string]string) error
+	// 'snapshotMeta' has metadata of the Volumesnaphot resource that is going to get created.
+	// 'contentMeta' has metadata of the VolumesnapshotContent content resource that is going to get created.
+	Clone(ctx context.Context, name, namespace string, waitForReady bool, snapshotMeta, contentMeta ObjectMeta) error
```

Comment:
```suggestion
	// 'cloneSnapshotMeta' has metadata of the Volumesnaphot resource that is going to get created.
	// 'contentMeta' has metadata of the VolumesnapshotContent content resource that is going to get created.
	Clone(ctx context.Context, name, namespace string, waitForReady bool, cloneSnapshotMeta, snapshotContentMeta ObjectMeta) error
```
Let's rename `contentMeta` -> `snapshotContentMeta` in all the occurrences